Solutions/CohesitySecurity/cohesity.json

```diff
@@ -1,22 +1,28 @@
 {
     "api_key": "<cohesity-helios-api-key>",
     "resource_group": "<resource-group-name>",
+    "resource_type": "Microsoft.OperationalInsights/workspaces",
```

Comment:
I don't see resource_type being used anywhere. Please remove it and add it a future PR when necessary.

Reply:
yes, not in use now, removed.